hrabsence/CRM/HRAbsence/BAO/HRAbsenceType.php

```diff
@@ -117,6 +77,59 @@ public static function create($params) {
   }
 
   /**
+   * Creates Activitu Types based on $params and $activityTypes
+   * @param  $activityTypes Array
+   * @param  $params        Array
+   * @param  $dr_cr         Boolean 'true' for 'debit' and 'false' for 'credit'
+   * @return Array          Created Activity Type as array
+   */
+  public static function createActivityType($activityTypes, $params, $dr_cr)
```

Comment:
Please always use camelCase for variables in PHP. The only exception is for variables mapped to db fields, in which snake_case should be used instead.

Also, avoid using abbreviations as much as possible. Explicit is always better than implicit. The $dr_cr variable, for example, can be renamed to $isDebit